ui/app/AppLayouts/Profile/views/AdvancedView.qml

```diff
@@ -291,11 +294,25 @@ SettingsContentBase {
                 anchors.leftMargin: Style.current.padding
                 anchors.rightMargin: Style.current.padding
                 visible: root.advancedStore.isWakuV2 && root.advancedStore.fleet != Constants.status_prod
-                text: qsTr("WakuV2 mode")
+                text: qsTr("WakuV2 options")
```

Comment:
Why can I not see this in the video? ;)

Reply:
not visible visible: root.advancedStore.isWakuV2 && root.advancedStore.fleet != Constants.status_prod

Reply:
you will see it if you change fleet eg wakuv2.prod
<img width="635" alt="w" src="https://user-images.githubusercontent.com/31625338/234811963-84a5b859-d780-4f51-bb13-9a17e2bd7c49.png">

Reply:
I see!